src/wmtk/TriMeshOperationExecutor.cpp

```diff
@@ -402,7 +497,88 @@ Tuple TriMesh::TriMeshOperationExecutor::split_edge()
     // return m_mesh.with_different_cid(m_operating_tuple, m_incident_face_datas[0].split_f0);
 }
 
+void TriMesh::TriMeshOperationExecutor::update_hash_in_map(TriMesh& child_mesh, const std::vector<long>& child_cell_ids_to_update_hash)
+{
+    long child_id = child_mesh.multi_mesh_manager.child_id();
+    auto child_hash_accessor = child_mesh.get_cell_hash_accessor();
+    for (auto child_cell_id : child_cell_ids_to_update_hash)
+    {
+        auto [t_child_old, t_parent_old] = MultiMeshManager::read_tuple_map_attribute(child_mesh.multi_mesh_manager.map_to_parent_handle, child_mesh, child_mesh.tuple_from_id(child_mesh.top_simplex_type(), child_cell_id));
```

Comment:
the point of the manager is to enable some easy functions in the Mesh class that simplify the notation... add functions to Mesh that do these sorts of reads rather than just call these massive manager names and have so many arguments